shared/slotutil/countdown.go

```diff
@@ -41,14 +41,14 @@ func CountdownToGenesis(ctx context.Context, genesisTime time.Time, genesisValid
 			ticker = time.NewTicker(time.Second)
 			secondTimerActivated = true
 		}
+		if timeRemaining >= time.Second {
+			log.WithFields(logFields).Infof(
+				"%s until chain genesis",
+				timeRemaining.Truncate(time.Second),
+			)
+		}
 		select {
 		case <-ticker.C:
-			if timeRemaining >= time.Second {
-				log.WithFields(logFields).Infof(
-					"%s until chain genesis",
-					timeRemaining.Truncate(time.Second),
```

Comment:
Instead of `timeRemaining.Truncate(time.Second)` change to `genesisTime.Sub(currentTime).Truncate(time.Second)` ?

Reply:
It will be kind of strange to calculate the remaining time first, then wait 1 second, and calculate it once again. So I though it is more reasonable to just place the log method before the ticker and to keep calculation in the single place.

Reply:
Makes sense. Thanks!